### DISCUSSION TITLE
Zowe 2.6v - How much time does it takes for Zowe to initialize

### DISCUSSION THREAD

**Discussion by Sam-Explorer** [zowe/community#2056] [Q&A]
Hello, 

In general, how much time does it take for Zowe STC to come up?
It takes upto 5 ~6 minutes whenever I had to start the STC, is that normal with Zowe?
Initially it was taking around 30+ minutes, which got improved after following - https://github.com/zowe/zowe-install-packaging/issues/2825 

Is there any changes that can be made to reduce the startup time?

Thanks

**Comment by balhar-jakub** [comment]
Hi Sam, 

In general I would say it's not unexpected. It depends on configuration and specific information about the system, but the least time to start I saw was 90 seconds due to the communication that needs to happen between the API ML services. 

And you are correct that the config manager related update, that you linked improved the situation massively. 

Best regards
Jakub

**Reply by Sam-Explorer** [reply]
Hi Jakub, 
It take's 5-6 mins to start, 90 seconds is still better, other than config manager. Any other code changes were done?

**Reply by balhar-jakub** [reply]
I believe the time was based on the environment. There were no other changes done for the specific customer to get to the 90 seconds.
